### ISSUE TITLE
Pipeware 0.3.25 on kernel 5.11.11. No input on external usb interface Presonus 1810c.

### ISSUE THREAD

**Issue by halturin** [NixOS/nixpkgs#122008]
I got an issue with my Presonus 1810c. There is no input present in my system. Googling the problem led me out to these bugs https://bugzilla.redhat.com/show_bug.cgi?id=1954660 and https://bugzilla.redhat.com/show_bug.cgi?id=1954522

pw-dump is enclosed 
[1810c.pw.dump.txt](https://github.com/NixOS/nixpkgs/files/6440698/1810c.pw.dump.txt)

**Comment by halturin** [comment]
just updated to Gnome 40.1 and pipewire 0.3.27 and the problem is still there.

**Comment by jansol** [comment]
Is this still a problem?

**Comment by halturin** [comment]
yep, I still don't see my presonus among the other input devices
![image](https://user-images.githubusercontent.com/118860/152560403-416f85f9-c49c-414d-b7fe-a8129f7c778b.png)

**Comment by jansol** [comment]
This looks like a configuration problem, specifically the interface being set to an output-only profile.
If you open pavucontrol and go to the Configuration tab, does it offer you any profiles with inputs for that card?

Not sure about what standard profiles are available for interfaces with that many ports, but if there are no profiles that work for you, you can always set the interface to Pro Audio profile and create a virtual sink and virtual source that target whatever AUX ports on the interface you want and provide a default sink with stereo/5.1/whatever channel layout to applications.

**Comment by halturin** [comment]
>and create a virtual sink and virtual source that target whatever AUX ports on the interface you want 

could you please give me a tip on how it can be done? is there any specific software for that?

**Comment by jansol** [comment]
The [pipewire wiki](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Virtual-Devices#examples) has some examples, as does the pipewire status tracking issue: #102547

**Comment by jansol** [comment]
Did you get it to work?
